package.json

```diff
@@ -25,7 +25,7 @@
     "build": "npm-run-all -p build:*",
     "build-workers": "cd ./client && pnpm run prebuild",
     "build:client": "cd ./client && pnpm run build",
-    "build:curriculum": "cd ./curriculum && pnpm run build",
+    "build:curriculum": "pnpm -F=curriculum run build && pnpm -F=client run build:external-curriculum",
```

Comment:
I gave this PR another look and was wondering if we could split the build process into separate commands: `build:internal-curriculum` and `build:external-curriculum`.

So this would be:

```json
"build:curriculum": "pnpm -F=curriculum run build:internal-curriculum && pnpm -F=client run build:external-curriculum"
```

And then in `predevelop`, we'd only need to run `build:internal-curriculum`.

https://github.com/freeCodeCamp/freeCodeCamp/blob/3fe392b375ab61342bd445448cc135027b63c3d5/package.json#L43

---

This could go in a separate PR, if we consider doing it.